docs/web/sdk-service-reference.mdx

```diff
@@ -0,0 +1,239 @@
+---
+title: Service Reference
```

Comment:
The name of this confused me at first, as it's so similar to `Directly Referencing Services`. Hmm not sure what to call it...

Reply:
I've never been a huge fan of the "Reference" naming pattern I went with...
"Component Reference", "Service Reference" etc. But I couldn't think of a better naming pattern for those articles.... 
They represent a repository of general technical info about a specific extension point in the SDK

Some other ideas, but also not sold on any of these -.-
"Component Objects", ...
"Service Class Reference", ...
"Events Resource, Components Resource,..."
"Service Guide"
"Components Technical Guide"


I'm also not opposed to changing "Directly Referencing Services" to something else either. The reason I went with "Directly Referencing" is because it described what the pattern accomplishes, and what distinguishes it from referencing services via commands and operations. Someone may not know what dependency injection is, but they will most likely know what "Directly Referencing" means.

Reply:
Maybe just `Service Injection`? That's how we refer to it when talking among ourselves.

Reply:
Changed to just "Service Injection"